src/cnc/models/config/resource.py

```diff
@@ -506,6 +510,36 @@ def queue_name(self):
         return queue_name
 
 
+class DynamoDBResourceSettings(BaseResourceSettings):
+    type: Literal["dynamodb"]
+    billing_mode: Optional[Literal["PROVISIONED", "PAY_PER_REQUEST"]] = (
+        "PAY_PER_REQUEST"
+    )
+    hash_key: Optional[str] = "UserID"
+    read_capacity: Optional[int] = 5
+    write_capacity: Optional[int] = 5
+    table_class: Optional[Literal["STANDARD", "STANDARD_INFREQUENT_ACCESS"]] = (
+        "STANDARD"
+    )
+    deletion_protection_enabled: Optional[str] = "false"
+
+    @property
+    def is_dynamodb(self):
+        return True
+
+    @property
+    def managed_environment_variables(self):
```

Comment:
now we need to do like f"DATABASE_ID_{self.name}" = self.dynamodb_table_output_id so that the vars will be set in the app

Reply:
Ajustments Commit: https://github.com/tmaior/cnc-fork/commit/9ed6e46909e2e43c43eee39bb138ed6d2df70b18